src/pages/new-lesson/NewLesson.styles.ts

```diff
@@ -0,0 +1,46 @@
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '24px',
+    py: '40px',
+    px: { sm: '36px', md: '72px' }
+  },
+  input: {
+    style: {
+      padding: 0
+    }
+  },
+  descriptionLabel: {
+    shrink: false,
+    sx: { typography: TypographyVariantEnum.Body1, top: -21 }
+  },
+  divider: {
+    color: 'primary.300'
+  },
+  titleInput: {
+    disableUnderline: true,
+    style: {
+      fontSize: '35px',
+      fontWeight: 500,
+      maxHeight: '35px',
+      marginTop: 0
+    }
+  },
+  descriptionInput: {
+    disableUnderline: true,
+    style: { fontSize: '16px', maxHeight: '16px', marginTop: 0 }
```

Comment:
same here, you can use our typography and pass it to sx

  <AppTextField
          InputProps={{disableUnderline: true}}
          sx={styles.descriptionInput}
        />